src/client/updater.py

```diff
@@ -141,13 +155,9 @@ def showSettings(self):
 @with_logger
 class UpdateChecker(QObject):
     gh_releases_url = Settings.persisted_property('updater/gh_release_url', type=str, default_value='https://api.github.com/repos/FAForever/client/releases?per_page=20')
+    updater_downgrade = Settings.persisted_property('updater/downgrade', type=bool, default_value=False)
```

Comment:
Why is it in 2 places at once? Is persisted_property capable of handling such a case (specifically does it update the other value when one of them changes)?